examples/website/icon/app.js

```diff
@@ -155,23 +155,20 @@ export class App extends Component {
   }
 
   render() {
-    const {viewState, controller = true, baseMap = true} = this.props;
+    const {mapStyle = 'mapbox://styles/mapbox/dark-v9'} = this.props;
 
     return (
       <DeckGL
         layers={this._renderLayers()}
         initialViewState={INITIAL_VIEW_STATE}
-        viewState={viewState}
-        controller={controller}
+        controller={{dragRotate: false}}
```

Comment:
why this is needed?

Reply:
This demo should only be viewed top-down.